app/custom_domain_validation.py

```diff
@@ -1,4 +1,12 @@
-from app.config import EMAIL_SERVERS_WITH_PRIORITY, EMAIL_DOMAIN
+from dataclasses import dataclass
+from typing import Optional
+
+from app.config import (
```

Comment:
If we import `from app import config` and then use it as `config.VAR_NAME` (for example `config.EMAIL_DOMAIN`) it will allow to change the config from tests if needed. Otherwise the value is stuck in when the file is initially loaded.

Reply:
The config variables are only used as default if no other variables are passed, so in the tests we can send the full configuration and we don't need to use the config as a side-effect, we specify which values are used

Reply:
fair enough

Reply:
Changed it